source/lib/cdk-textract-stack.ts

```diff
@@ -103,22 +119,17 @@ export class CdkTextractStack extends cdk.Stack {
       {
         websiteIndexDocument: "index.html",
         cors: [corsRule],
-        encryption: BucketEncryption.S3_MANAGED
+        encryption: BucketEncryption.S3_MANAGED,
+        serverAccessLogsBucket: logsS3Bucket,
+        serverAccessLogsPrefix: "clientAppS3Logs"
```

Comment:
Same comment on prefix name as for document-s3-bucket

Reply:
Updated prefix